pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -163,6 +163,26 @@ var _ = Describe("nic hotplug on virt-launcher", func() {
 			dummyDomain(networkName),
 			libvirtClientResult{expectedAttachedDevices: 1},
 		),
+		Entry("VMI with 1 network, and link state down (with the pod interface ready), not present in the domain",
+			func() *v1.VirtualMachineInstance {
```

Comment:
nit: Consider replacing this anonymous function with `libvmi` as was performed in a following commit in order to improve readability.

Same bellow.

Reply:
I still see it.